Sources/QuaternionModule/Quaternion.swift

```diff
@@ -339,6 +396,17 @@ extension Quaternion where RealType: BinaryFloatingPoint {
 
 // MARK: - Conformance to Hashable and Equatable
 extension Quaternion: Hashable {
+  /// Returns a Boolean value indicating whether two values are equal.
+  ///
+  /// Equality is the inverse of inequality. For any values *a* and *b*,
+  /// `a == b` implies that `a != b` is `false`.
```

Comment:
This is true for anything conforming to `Equatable`, so I don't think that we need to spell it out here; is there a specific motivation for adding it?

Reply:
No specific reason. I thought it might be nice to have same comments as `==` from `Equatable`. I removed it.